ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/V2/LedgerSeq.hs

```diff
@@ -81,7 +81,7 @@ data LedgerTablesHandle m l = LedgerTablesHandle {
     -- | Costly read all operation, not to be used in Consensus but only in
     -- snapshot-converter executable.
   , readAll            :: !(m (LedgerTables l ValuesMK))
-  , pushDiffs          :: !(LedgerTables l DiffMK -> m ())
+  , pushDiffs          :: !(forall mk. l mk -> l DiffMK -> m ())
```

Comment:
This would benefit from some docs explaining about how the two ledger state should be related